### PR TITLE
Chore/adjust admin dashboard logic

### PR DESCRIPTION
**Pull request by willblake01**
This is the companion PR to go along with #166 that updates the admin dashboard to reflect cancellations in the Participation table.

*As always I'm eager for feedback, let me have it!

### CODE REVIEW

File: app/views/admin/dashboard/show.html.haml [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
+      = column_chart Participant.with_payment.not_cancelled.not_archived.group_by_week(:created_at, range: 9.months.ago..Time.now).count
       .card-footer
-        %p.text-muted Shows new paid-up participants (race-signups) grouped by week from 3 weeks ago.
+        %p.text-muted Shows new paid-up participants (race-signups) net of cancellations grouped by week from 9 months ago.
   .col-6
     .card
       .card-header.bg-success
         %h4.text-white Payments
-      = line_chart Payment.group_by_week(:created_at, range: 3.weeks.ago.beginning_of_day..Time.now).sum('amount_charged_in_cents / 100')
+      = line_chart [{ name: "Payments", data: Payment.group_by_week(:created_at, range: 9.months.ago.beginning_of_day..Time.now).sum('amount_charged_in_cents / 100') }, { name: "Refunds", data: Refund.group_by_week(:created_at, range: 9.months.ago.beginning_of_day..Time.now).sum('amount_in_cents / 100') }], prefix: "$"
```
Comment (willblake01) [inline app/views/admin/dashboard/show.html.haml:16]: @ColinW520 I couldn't get this to work when refactored for line length for some reason. The text wrap in my IDE makes it look much tidier but am I missing something that's not allowing me to refactor this to be more readable?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -36,5 +36,6 @@
 //= require toastr.min.js
 //= require ekko_lightbox.min.js
 //= require chartkick
+//= require Chart.bundle
```
Comment (ColinW520) [inline app/assets/javascripts/application.js:39]: Where is Chart.bundle coming from? What does this do for us that adjusts dashboard logic?

Totally cool to includes it ; but, we should be sure not to mix adding new features with refactors/fixes

File: app/views/admin/dashboard/show.html.haml [hunk 3]
```diff
@@ -35,7 +35,7 @@
                 = link_to event_race_participants_path(race.event, race), target: "_blank" do
                   %i.fa.fa-external-link-alt{"data-placement" => "right", "data-toggle" => "tooltip", :title => "See the start list"}
                   = race.name_with_event
-              %td= race.paid_participants_count
+              %td= race.participants.for_start_list.size
               %td= race.participants_cap
-              %td= race.participants_cap - race.paid_participants_count
+              %td= race.participants_cap - race.participants.for_start_list.size
```
Comment (ColinW520) [inline app/views/admin/dashboard/show.html.haml:40]: This is the part of this PR that _fixes_ -- the rest seems to be additive. Split the two up! It's a great way to learn to isolate your changes to that which is most pertinent to the issue being resolved.

Comment (willblake01) [reply app/views/admin/dashboard/show.html.haml:40]: I had this thought after I created the PR. I'll split them up, I knew better 😔

**Pull request merged**